articles/active-directory-b2c/tutorial-single-page-app-webapi.md

```diff
@@ -29,17 +29,16 @@ In this tutorial, you learn how to:
 ## Prerequisites
 
 * Complete the steps and prerequisites in [Tutorial: Enable authentication in a single-page application using Azure Active Directory B2C](tutorial-single-page-app.md).
-* Visual Studio 2019 or later, or Visual Studio Code
-* .NET Core 2.2 or later
-* Node.js
+* [Visual Studio Code](https://code.visualstudio.com/)
+* [Node.js](https://nodejs.org/en/download/)
 
 ## Add a web API application
 
 [!INCLUDE [active-directory-b2c-appreg-webapi](../../includes/active-directory-b2c-appreg-webapi.md)]
 
 ## Configure scopes
 
-Scopes provide a way to govern access to protected resources. Scopes are used by the web API to implement scope-based access control. For example, some users could have both read and write access, whereas other users might have read-only permissions. In this tutorial, you define both read and write permissions for the web API.
+Scopes provide a way to govern access to protected resources. Scopes are used by the web API to implement scope-based access control. For example, some users could have both read and write access, whereas other users might have read-only permissions. In this tutorial, you define a read permission for the web API.
 
 [!INCLUDE [active-directory-b2c-scopes](../../includes/active-directory-b2c-scopes.md)]
```

Comment:
Since this include is pulled in by several articles, recommend that we leave demo.write in there (as you've done) for now.

```console
$ grep -rin --include=*.md "active-directory-b2c-scopes.md"
tutorial-single-page-app-webapi.md:44:[!INCLUDE [active-directory-b2c-scopes](../../includes/active-directory-b2c-scopes.md)]
add-web-application.md:59:[!INCLUDE [active-directory-b2c-scopes](../../includes/active-directory-b2c-scopes.md)]
tutorial-web-api-dotnet.md:71:[!INCLUDE [active-directory-b2c-scopes](../../includes/active-directory-b2c-scopes.md)]
tutorial-desktop-app-webapi.md:40:[!INCLUDE [active-directory-b2c-scopes](../../includes/active-directory-b2c-scopes.md)]
```

Reply:
I understand. I'll clarify this in the source code